frontend/src/components/Navbar/Navbar.css

```diff
@@ -0,0 +1,30 @@
+.navbar {
+  margin-left: 5rem;
+  background-color: white;
+  justify-content: center;
+  align-items: center;
+}
+
+.navbar-items {
+  margin-left: 2rem;
+  border: none;
+  background-color: white;
+}
+
+.navbar-items > svg {
+  margin: 0rem;
+  padding: 0em;
```

Comment:
When the value of a property is `0`, the unit does not matter. `0px` is the same as `0rem`, which is the same as `0%` and so on...

I believe there already is a style that sets the default `margin`/`padding` of everything to `0`, so this is irrelevant. Remove this.